### ISSUE TITLE
[Feature request] Comments sort type

### ISSUE THREAD

**Issue by pitrisss** [sheodox/alexandrite#30]
It would be nice, if we could define what comment sorting we would have on default. Not sure how complicated would this be though.

**Comment by dtantono** [comment]
Yes, wefwef and memmy have this feature. I hope Alexandrite can do the same
